decidim-blogs/spec/forms/decidim/blogs/admin/post_form_spec.rb

```diff
@@ -13,6 +13,8 @@ module Admin
         end
 
         let(:current_organization) { create(:organization) }
+        # let(:current_user) { create :user, organization: current_organization }
+        # let(:user_group) { create(:user_group, :verified, organization: current_organization) }
```

Comment:
```suggestion
        let(:current_user) { create :user, organization: current_organization }
        let(:user_group) { create(:user_group, :verified, organization: current_organization) }
        let(:author_id { "current_organization" }
```

In here you will need to test for several `author_id` (now called `user_group_id` in `post_form.rb`). Best way to do it is to assign a variable that you can change between contexts.

Then, down on the `attributes` variable, add the author_id as part of the params:
```ruby
        let(:attributes) do
          {
            "post" => {
              "title" => title,
              "body" => body,
              "author_id" => author_id
            }
```

Reply:
Done